src/db_impl.cc

```diff
@@ -1099,49 +1100,66 @@ Options TitanDBImpl::GetOptions(ColumnFamilyHandle* column_family) const {
   return options;
 }
 
+Status TitanDBImpl::ExtractTitanCfOptions(
```

Comment:
As it never returns non-okay status, why not use `changed` as return value

Reply:
The caller below seems not checking the status but just the `changed` parameter